samples/boards/stm32/power_mgmt/suspend_to_ram/dts/bindings/test-spi-loopback.yaml

```diff
@@ -0,0 +1,13 @@
+#
+# Copyright (c) 2022 Kumar Gala <galak@kernel.org>
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+description: |
+    This binding provides resources required to build and run the
+    tests/drivers/spi/spi_loopback test in Zephyr.
+
```

Comment:
"under power management conditions"

Reply:
probably requires an update  of the README.rst  to explain that new configuration